README.md

```diff
@@ -18,6 +18,12 @@ It may be advisable to make sure all actions and channels converge to email.
 * on github check assigned issues, reviews, pull requests (this is all reflected in notifications, and other fields in the github page header)
 * communicate by mattermost and slack
 
+Note that if the PR is not draft, someone will need to select reviewers. If the PR is made by internal developer, only this developer will select reviewers. The PR is made by external developer - any relevant internal developer will select the reviewers.
```

Comment:
If the PR is made external developer... ?

Reply:
eh what? It says? I put a list to be more clear.